webserver/src/webserver.cpp

```diff
@@ -38,7 +38,10 @@ po::options_description commandLineArguments()
          "Logging level of the parser. Possible values are: debug, info, warning, "
          "error, critical")
         ("jobs,j", po::value<int>()->default_value(4),
-         "Number of worker threads.");
+         "Number of worker threads.")
+        ("log-target", po::value<std::string>(),
```

Comment:
I think this would be better placed after `loglevel`. The naming convention should also be unified, either `log-level` or `logtarget`.

Reply:
Changed it to "logtarget".